src/popup/components/TransferSend/TransferSendRecipient.vue

```diff
@@ -128,15 +148,26 @@ export default defineComponent({
 @use '@/styles/variables' as *;
 
 .transfer-send-recipient {
-  .scan-button {
-    color: $color-white;
-    display: block;
-    width: 32px;
-    height: 24px;
-  }
-
   .status {
     margin-top: 9px;
   }
+
+  .buttons {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+    gap: 4px;
+    opacity: 0.75;
```

Comment:
Why the buttons now has slightly lower opacity? I guess this is a mistake.

Reply:
It's according to figma